internal/configtxgen/encoder/encoder.go

```diff
@@ -329,6 +329,8 @@ func NewOrdererOrgGroup(conf *genesisconfig.Organization) (*cb.ConfigGroup, erro
 
 	if len(conf.OrdererEndpoints) > 0 {
 		addValue(ordererOrgGroup, channelconfig.EndpointsValue(conf.OrdererEndpoints), channelconfig.AdminsPolicyKey)
+	} else {
+		return nil, errors.Errorf("orderer endpoints for organization %s are missing and must be cofigured", conf.Name)
```

Comment:
Do we really want to make OrdererEndpoints required? For example, could an OrdererOrg be added to the channel initially without OrdererEndpoints (without any orderer nodes), with the intent for that organization to later add their own nodes.

Reply:
Dave, this is `encoder.go`. It's only used when creating genesis blocks.